crates/parcel_core/src/types/source.rs

```diff
@@ -3,17 +3,17 @@ use std::path::PathBuf;
 use serde::Deserialize;
 use serde::Serialize;
 
-#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, Eq, Hash, PartialEq, Serialize)]
 pub struct Location {
-  pub line: u32,
-  pub column: u32,
+  pub line: usize,
```

Comment:
`usize` is architecture dependant & on 64-bit architecture it won't fit JS numbers, requiring big-int

I reckon this breaks passing the value as a napi object ; not sure if this at the moment is going to serialize to JSON in the bridge and hence avoid that.

Reply:
I see, the issue is that most other crates store lines and columns as `usize` so I figured I'd make it consistent with that since it's a pain to convert them